src/aktualizr_info/CMakeLists.txt

```diff
@@ -40,6 +42,7 @@ add_test(NAME aktualizr-info-images-target
 set_tests_properties(aktualizr-info-images-target PROPERTIES
     PASS_REGULAR_EXPRESSION "image targets.json content:\n\n\\{.*\\}")
 
-aktualizr_source_file_checks(${AKTUALIZR_INFO_SRC} ${AKTUALIZR_INFO_HEADERS})
-
+set(AKTUALIZR_INFO_ALL_CHECKS ${AKTUALIZR_INFO_SRC} ${AKTUALIZR_INFO_HEADERS} ${TEST_SOURCES})
+list(REMOVE_DUPLICATES AKTUALIZR_INFO_ALL_CHECKS)
```

Comment:
idea: we could actually move `REMOVE_DUPLICATES` in `aktualizr_source_file_checks` since it seems to be a common problem (`add_aktualizr_test` does it for us and if we have conditionally-run tests it becomes a headache).

Reply:
Yeah, not a bad idea at all. I'll give that a try.

Reply:
Done. Now ready for re-review.